docs-source/spring/content/development/cli/_index.md

```diff
@@ -957,6 +1011,208 @@ GraalVM Compile Commands
+
+OPTIONS
+       --username String
+       The name you assign to the user during creation. This is the user’s login for the CLI. The name must be unique across all users in the platform and cannot be changed.
+       [Mandatory]
+
+       --roles String
+       The user's role within the platform. A user must have up to three possible roles provided in a comma-separated list. [ROLE_ADMIN,ROLE_USER].
+       [Optional, default = ROLE_USER]
+
+
+...
+
+```
+
+Example:
+
+```cmd
+oractl:>user create --username user1 --roles ROLE_ADMIN,ROLE_USER
+? password ****************
+obaas-cli [user create]: User [user1] as successfully created.
+```
+
+#### Delete User
+
+Use the `user delete` command to remove users from the platform.
+
+```cmd
+oractl:>help user delete
+NAME
+       user delete - Delete a user in your platform.
+
+SYNOPSIS
+       user delete [--username String] --id int --help
+
+OPTIONS
+       --username String
+       The username you want to delete.
+       [Mandatory]
+
+       --id int
+       The user id from the user you want to delete.
+       [Optional, default = 0]
+
+...
+
+```
+
+Example:
+
+```cmd
+oractl:>user delete --username user1
+obaas-cli [user delete]: User [user1] as successfully deleted.
+````
+
+#### Obtain User details
+
+Use the `user get` command to obtain the user details registered on the platform.
+
+```cmd
+oractl:>help user get
+NAME
+       user get - Gets the specified user’s information.
+
+SYNOPSIS
+       user get [--username String] --help
+
+OPTIONS
+       --username String
+       The username of the user.
+       [Mandatory]
+
+...
+
+```
+
+Example:
+
+```cmd
+oractl:>user get --username obaas-admin
+╔══╤═══════════╤═══════════════════════════════════════╗
+║Id│Username   │Roles                                  ║
+╠══╪═══════════╪═══════════════════════════════════════╣
+║2 │obaas-admin│ROLE_ADMIN,ROLE_USER                   ║
+╚══╧═══════════╧═══════════════════════════════════════╝
+```
+
+#### Change User Roles
+
+Use the `user change-roles` command to change the roles from a specific user registered on the platform.
+
+```cmd
+oractl:>help user change-roles
+NAME
+       user change-roles - Change the roles from the specified user.
+
+SYNOPSIS
+       user change-roles [--username String] --roles String --help
+
+OPTIONS
+       --username String
+       The name you assign to the user during creation. This is the user’s login for the CLI.
+       [Mandatory]
+
+       --roles String
+       The user's role within the platform. A user must have up to three possible roles provided in a comma-separated list. [ROLE_ADMIN,ROLE_USER].
+       [Optional, default = ROLE_USER]
+
+...
+
+```
+
+Example:
+
+```cmd
+oractl:>user change-roles --username user1 --roles ROLE_USER
+obaas-cli [user change-roles]: User [user1] roles were successfully updated.
+```
+
+#### Change User Password
+
+Use the `user change-password` command to change the password from a specific user registered on the platform.
+
```

Comment:
A user password can be changed by a user with ROLE_ADMIN or the user it self can change It's password

Reply:
"A user is allows to change their own password only"

Reply:
its -> their